tests/console/aide_check.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use testapi;
+use strict;
+
+# test for basic function of aide. Check different between aide.db and file system
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+    sleep 2;
+    validate_script_output "zypper -n in aide", sub { m/(Installing.*)|(.*already installed)/ };
+    sleep 2;
+    assert_script_run "cp /etc/aide.conf /etc/aide.conf.bak";
+    sleep 2;
```

Comment:
why would you sleep 2 after an asserted cp? just remove all sleep 2 - and please tell me where you copy&pasted it from, so can we fix that test too